packages/forms/src/UIForm/utils/validation.js

```diff
@@ -70,3 +173,68 @@ export function isValid(schema, errors) {
 
 	return true;
 }
+
+/**
+ * Filter the errors on array which items indexes are between a range
+ * This returns only the errors keys.
+ * @param errors The errors map
+ * @param arrayKey The array key
+ * @param minIndex The min item index (INCLUDED)
+ * @param maxIndex The max item index (EXCLUDED)
+ */
+export function filterArrayErrorsKeys(errors, arrayKey, minIndex, maxIndex) {
+	const minArrayIndexKey = Number.isInteger(minIndex) && arrayKey.concat(minIndex).toString();
+	const maxArrayIndexKey = Number.isInteger(maxIndex) && arrayKey.concat(maxIndex).toString();
+
+	return Object.keys(errors)
+		.filter(errorKey => errorKey.startsWith(arrayKey))
+		.filter(errorKey => !minArrayIndexKey || errorKey >= minArrayIndexKey)
+		.filter(errorKey => !maxArrayIndexKey || errorKey < maxArrayIndexKey);
```

Comment:
With you example, you'll potentially loop `3 * errors.length` times.

I think that the following code is not less clear, but I let you decide :

```javascript
With you example, you'll potentially loop `3 * errors.length` times.

I think that the following code is not less clear, but I let you decide :

```javascript
return Object.keys(errors)
		.filter((e) => {
			const isOnTargetArray = ...;
			const isAfterMinIndex = ...;
			const isBeforeMaxIndex = ...;
			
			return isOnTargetArray && isAfterMinIndex && isBeforeMaxIndex;
		});
```